virtme/commands/run.py

```diff
@@ -1638,7 +1707,10 @@ def get_net_mac(index):
         if args.server == "console":
             console_server(args, qemu, arch, qemuargs, kernelargs)
         elif args.server == "ssh":
-            ssh_server(args, arch, qemuargs, kernelargs)
+            # Turn on vsock if available
+            ssh_server(
+                args, arch, qemuargs, kernelargs, vsock=can_use_ssh_over_vsock(args)
+            )
```

Comment:
out of curiosity: is this style impose by the new tools?

Maybe "better" with:

```suggestion
            ssh_server(args, arch, qemuargs, kernelargs,
                       vsock=can_use_ssh_over_vsock(args))
```

Reply:
See new commits.